chips/arty_e21_chip/src/chip.rs

```diff
@@ -134,11 +121,8 @@ impl<'a, I: InterruptService<()> + 'a> ArtyExx<'a, I> {
             addi t0, t0, %lo(_start_trap)
```

Comment:
I think a `li t0, _start_trap`, which will expand to a `lui` and `addi`, is a bit nicer.